### PR TITLE
添加vip支持aap功能介绍

### PR DESCRIPTION
**Pull request by zcq98**
- [ ] Make sure you have followed the [Document Convention](../docs/reference/document-convention.md).

添加vip支持aap功能介绍，添加英文文档

https://github.com/kubeovn/kube-ovn/pull/3359

### CODE REVIEW

File: docs/advance/vip.md [hunk 1]
```diff
@@ -84,3 +88,99 @@ spec:
...
+spec:
+  subnet: ovn-default
+  namespace: default
+  selector:
+    - "app: aap1"
+```
+
+VIP 同样支持固定地址和随机地址的分配，分配方式如上文所述。
+
+- `namespace`: 需显式地指定命名空间，VIP 仅允许相同命名空间的资源开启 AAP 功能。
```
Comment (bobz965) [inline docs/advance/vip.md:108]:
```suggestion
- `namespace`: AAP 场景下，vip 需显式地指定命名空间，VIP 仅允许相同命名空间的资源开启 AAP 功能。
```

File: docs/advance/vip.md [hunk 2]
```diff
@@ -84,3 +88,99 @@ spec:
...
+- `selector`: AAP 的 Pod 的节点选择器，格式和 Kubernetes 中的 NodeSelector 格式相同。
```
Comment (bobz965) [inline docs/advance/vip.md:109]:
```suggestion
- `selector`: AAP 场景下，用于选择 vip 所附属的 Pod 的节点选择器，格式和 Kubernetes 中的 NodeSelector 格式相同。
```

Comment (bobz965) [reply docs/advance/vip.md:109]: 需要补充下说明，该功能支持多网卡场景

**Review by bobz965 — APPROVED** [review]

**Comment by oilbeater** [comment]
Lint issues should be fixed

**Pull request merged**